spec/spec_helper.rb

```diff
@@ -16,4 +18,12 @@
   config.warnings = true
 
   Dir[File.expand_path('support/**/*.rb', __dir__)].each { |f| require f }
+
+  config.before(:suite) do
+    FileUtils.mkdir_p("tmp_#{Process.pid}")
```

Comment:
We can use the system temporary directory rather than add churn to the current working directory. This is often a ram disk and gets cleaned automatically on restart.

https://ruby-doc.org/stdlib-3.1.2/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir
https://ruby-doc.org/stdlib-3.1.2/libdoc/tmpdir/rdoc/Dir.html#method-c-mktmpdir